.gitignore

```diff
@@ -146,7 +146,6 @@ Desktop.ini
 .dbus/
 repos/
 repos-download/
-/compass
```

Comment:
Not sure if we want this? See #897.

Reply:
I think it can be removed as we don't use the root directory `compass` (anymore), at least I am not aware of it. Maybe @krusche can say more about it?

Reply:
it's ok